minethd.cpp

```diff
@@ -33,7 +33,11 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
+#ifdef __MINGW64__
+	SetThreadAffinityMask(&h, 1ULL << cpu_id);
```

Comment:
Not the case with the MinGW!
The test actually failed when value is passed to SetThreadAffinityMask (**void* hThread**, DWORD_PTR dwThreadAffinityMask); on MinGW.

If HANDLE = void *
Handle can accept:
 - var, where var is T* var
 - &var, where var is T var